File: python/math/calculate/src/formula/formula.py
#!/usr/bin/python3
import os
import json

def Run(numberOne, numberTwo, operation):
    input_json = json.dumps({'number_one': numberOne, 'number_two': numberTwo})

    if operation == "sum":
        stdin_cmd = f"echo '{input_json}' | rit python math sum numbers --stdin"

    elif operation == "multiplication":
        stdin_cmd = f"echo '{input_json}' |  rit python math multiply numbers --stdin"

    else:
        print("Unexpected operation")

    os.system(f'{stdin_cmd}')

File: node/math/calculate/src/formula/formula.js
const clc = require("cli-color")

function execute(command) {
    const { exec } = require('child_process');

    exec(command, (error, stdout, stderr) => {
        if (error) {
          console.error(`exec error: ${error}`);
          return;
        }
        console.log(`stdout: ${stdout}`);
        console.error(`stderr: ${stderr}`);
    })
}

function Run(numberOne, numberTwo, operation) {
    console.log("Number operations")

    if(operation === 'sum'){
        execute(`echo '{ "number_one": "'${numberOne}'", "number_two": "'${numberTwo}'", "operation": "'${operation}'" }' | rit math sum numbers --stdin`)
    } else if (operation === 'multiply'){
        execute(`echo '{ "number_one": "'${numberOne}'", "number_two": "'${numberTwo}'", "operation": "'${operation}'" }' | rit math multiply numbers --stdin`)
    } else {
        console.log(clc.red("(Invalid Input) Use sum or multiply!"))
    }
}

const formula = Run
module.exports = formula
